exporter/datadogexporter/internal/translator/dimensions.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package translator
+package translator // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/translator"
```

Comment:
This was fixed on #6502, we can remove from here

Reply:
Neat, sorry I was trying to resolve the linting issues

Reply:
No worries, sorry that you had to fix the lint issues in the first place :)